hub.py

```diff
@@ -299,7 +307,13 @@ async def check_hub_health(self, test_notebook_path, service_api_token):
         before declaring it a failure. If any of these steps fails, immediately halt further
         deployments and error out.
         """
-        hub_url = f'https://{self.spec["domain"]}'
+
+        hub_org_domain = (
+            self.spec["domain"]
+            if isinstance(self.spec["domain"], str)
```

Comment:
We should probably also use the same logic as earlier (auth0.domain overriding domain) here.